docs/guidance/build/data_collection_tools.md

```diff
@@ -0,0 +1,104 @@
+
+The OCDS Helpdesk can:
+
+* Provide guidance on using the template to collect data
+* Help you to extend or adapt the template to meet your requirements
+* Help you to analyze the data that you collect
+
+## OpenContractR
+
+[OpenContractR](https://github.com/patxiworks/opencontractr) is a WordPress plugin for collecting and publishing OCDS data.
+
+The plugin adds an interface for entering data, a contracts search function, visualizations and an OCDS format JSON API.
+
+OpenContractR can be added to a new or existing WordPress site. There are many WordPress hosting providers to choose from if you do not already use WordPress.
+
+Consider using OpenContractR if:
+
+* You already have a WordPress website, or you have the capacity to set one up
+* Your users have access to reliable internet connections
+* Data entry will be done by many different users
+* You want to minimize the work required to collate data
+* You want users to be able to edit or update already entered data
+* You want an online search interface for your data
+
+Read more about OpenContractR in its [introduction](https://drive.google.com/file/d/18WHnQcnA6oESZtcZgS4rgaBLjd8BKvbM/view). If you’re interested in using OpenContractR please get in touch with the OCDS Helpdesk.
+
+## Contrataciones Abiertas tool
+
+[Contrataciones Abiertas](https://github.com/INAImexico/Contrataciones_abiertas_v2) is a tool developed by the Mexican government and it is made up of two main modules:
+
+* Information capture module
+* Visualization module
+
+The objective of the tool is to enable the monitoring of contracting processes by using and producing OCDS data.
+
+Consider using Contrataciones Abiertas if:
+
+* You have the resources and technical capacity to [set up the tool](https://github.com/INAImexico/Contrataciones_abiertas_v2/raw/master/Manual%20de%20instalaci%C3%B3n.docx)
+* Data entry will be done by different users
+* You want to validate your data before publication
+* You want to minimize the work required to collate and publish data
+* You want a [dashboard and visualizations](https://www.open-contracting.org/2020/06/12/mexicos-inai-launches-new-open-source-tool-to-upload-and-visualize-open-contracting-data/) of your data
+* Spanish is your (and your users') main language
+
+The tool is open source and is documented in Spanish [here](https://github.com/INAImexico/Contrataciones_abiertas_v2). You can also find an introduction to the tool in [our blog](https://www.open-contracting.org/2020/06/12/mexicos-inai-launches-new-open-source-tool-to-upload-and-visualize-open-contracting-data/).
+
+## What next?
+
+However you choose to collect and structure your data, you should publish it online using an [open license](../../publish/#license-your-data) so that others can make use of it.
+
+Consider how you will make it easy for users to discover the data you have published. For example, by publishing it on an existing procurement portal, on your organization’s website, or an open data portal.
```

Comment:
Leave the licensing content out. It's covered in the publish step.

```suggestion
However you choose to collect and structure your data, consider how you will make it easy for users to discover the data you have published. For example, by publishing it on an existing procurement portal, on your organization’s website, or an open data portal.
```

Reply:
Replaced with a link to the relevant guidance in https://github.com/open-contracting/standard/pull/1061/commits/004745d3dd8fec737cc28690a4bf8f64c45ebdce